### DISCUSSION TITLE
Render parking fee=no

### DISCUSSION THREAD

**Discussion by ThomasMeister123** [osmandapp/OsmAnd#14876] [Q&A]
How to change the icon in render.xml for parking fee=no?
I tried this (first try with an other icon [taxi]), but it doesnt work:

```
<case minzoom="16" tag="amenity" value="parking" icon="amenity_parking" iconVisibleSize="38" iconOrder="151">
	
	<case additional="parking=multi-storey" icon="parking_multi_storey">
		<case additional="fee=yes" icon="amenity_parking_multi_storey_paid"/>
	</case>
	<case additional="parking=underground" icon="parking_underground">
		<case additional="fee=yes" icon="amenity_parking_underground_paid"/>
	</case>
	<case additional="fee=yes" icon="amenity_parking_paid"/>
	<apply_if additional="access=customers" icon="amenity_parking_private"/>
	<apply_if additional="access=private" icon=""/>
	<apply_if additional="access=no" icon=""/>
	<apply_if additional="fee=no" icon="amenity_taxi"/>
</case>
```

**Comment by thompehw** [comment]
Speaking just as a fellow user, it looks to me that fee=no is not included in rendering_types.xml so is not available to the renderer. The best you could do would be to render an absence of fee=yes. I found some documentation on rendering_types [here](https://osmand.net/docs/technical/map-creation/create-offline-maps-yourself/#custom-vector-map-tags).

However, fee=no is included in poi_types.xml which explains why you can create a POI search for parking with fee=no. E.g. type "parking" into the general search box, select Parking (Transportation) from the results, tap the filter icon at the top right (below the X), select Fee No and Apply Filters, then SHOW PARKING ON THE MAP. Also, while on the filters screen if you tap the three dots at the top right you can save with a name (E.g. "Free Parking") that you can quickly access under Search>Categories (the order of which can be rearranged) in future.

I look forward to being corrected!

**Comment by ThomasMeister123** [comment]
Thank you! Nice to see that the Filter fee=no is now available in tje newest version. I dont updated until now ("never change a running System" 😊). But in this case I will load the newest Version

**Comment by ThomasMeister123** [comment]
OK, now I see: it's useless again. The "stupid" filters allow fee=no and access=customers. BUT, I need all free parking WITHOUT the parkingplaces with access=customers (or without the tag "access") . But it's not possible 👎
